codalab/common.py

```diff
@@ -101,10 +101,12 @@ class State(object):
     RUNNING = 'running'   # Actually running
     READY = 'ready'       # Done running and succeeded
     FAILED = 'failed'     # Done running and failed
+    KILLED = 'killed'     # Killed by user
```

Comment:
To me, it makes sense to differentiate between a user aborted run and a run that failed and dies on its own. Too many times I've panicked before realizing a run shown as 'failed' was actually something I killed a while ago.

Perhaps the bigger reason is any signal that can give clues as to whether a failure is the user's fault or potentially our platform's fault is valuable in a practical sense.